modules/openapi-generator/src/main/resources/swift5/Extensions.mustache

```diff
@@ -65,7 +65,18 @@ extension Date: JSONEncodable {
     func encodeToJSON() -> Any {
         return CodableHelper.dateFormatter.string(from: self)
     }
-}{{/useVapor}}{{#generateModelAdditionalProperties}}
+}
+
+extension JSONEncodable where Self: Encodable {
+    func encodeToJSON() -> Any {
+        let encoder = CodableHelper.jsonEncoder
+        guard let data = try? encoder.encode(self) else {
+            fatalError("Could not encode to json: \(self)")
+        }
+        return data.encodeToJSON()
+    }
+}
+{{/useVapor}}{{#generateModelAdditionalProperties}}
```

Comment:
@0x0c just a small nit, could you please make this in one line to improve the generated code?
```
}{{/useVapor}}{{#generateModelAdditionalProperties}}
```

Reply:
Done with https://github.com/OpenAPITools/openapi-generator/pull/11202/commits/8b1529fbd02a9b089e08b68968286b2f00404941

Reply:
Thanks, could you please update the sample projects also?

Reply:
OK!

Reply:
The last commit brought a lot of undesired changes, could you please revert it?
Maybe you need to first merge the master branch into yours and then run the following commands.
```
./mvnw clean package 
./bin/generate-samples.sh
./bin/utils/export_docs_generators.sh
```

Reply:
Done with https://github.com/OpenAPITools/openapi-generator/pull/11202/commits/5a1661b918d003dd80a6a1e1c434f0c3de4bf1c5 and https://github.com/OpenAPITools/openapi-generator/pull/11202/commits/51fe33fde2afc422692f8937749a769fbab56a17

Reply:
Could you please explain why some enums are not JSONEncodable?
Is there any issue that you found?

Reply:
Since enum value that confirms `RawRepresentable` a.k.a primitive value is already confirm `JSONEncodable` [here](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/swift5/Extensions.mustache#L24) and [here](), it seems to that `modelEnum.mustache` and `modelInlineEnumDeclaration.mustache` should not confirm `JSONCodable`.

Reply:
You are right, thanks 👍